### ISSUE TITLE
頻度,発音記号,意味,発音のDBを作る

### ISSUE THREAD

**Issue by hrk-htnk** [YHU-team18/Read-files-and-learn-words#11]
# 概要
Wordに関連した頻度,発音記号,意味,発音のDBを作成する
# 目的
Wordから意味,発音記号,発音を持ってこれるようにするため
# 提案内容
頻度,発音記号,意味,発音のテーブルを作成し,csvファイルからpostgreSQLに格納
# タスク

- まずどのようなファイル形式で保存されているかを確認
- 頻度,発音記号,意味,発音のテーブル作成
- csvファイルなら,それをpostgreSQLに格納

- [ ] #24
- [x] #23
- [ ] Djangoのモデルを定義する., 特に一対一対応
- [ ] 複数のDBを扱う(それぞれどのように保存するか?, 等を調べる)

**Comment by makinzm** [comment]
- [ ] [事前データのある,Dockerfileを書く](https://amateur-engineer.com/docker-compose-postgresql/)
- [ ] textをposgreに変換する
- [ ] Djangoのモデルを定義する., 特に一対一対応

**Comment by makinzm** [comment]
ありがとうございます！

これはdocker fileというよりかは、srcの中でという解釈であってますかね？

午後あたり確認します🙇‍♂️

**Comment by ROBO358** [comment]
> これはdocker fileというよりかは、srcの中でという解釈であってますかね?

そうですね。`loaddata`するという事自体は`Dockerfile`や`docker-compose.yml`内で書く必要があるとは思いますが、基本的にはsrc内にDBに保存するデータを持っておくという感じになると思います。

**Comment by makinzm** [comment]
https://github.com/makinzm/EJDict
